src/apps/SafeConnect/App.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { useState, useEffect } from "react";
+import jsQr from "jsqr";
+import styled from "styled-components";
+import format from "date-fns/format";
+
+import {
+  TextField,
+  Button,
+  Text,
+  Title,
+  Icon,
+  Loader,
+} from "@gnosis.pm/safe-react-components";
+
+import { blobToImageData } from "./utils";
+import { Wrapper } from "./components/layout";
+import WCClientInfo from "./components/WCClientInfo";
+import useWalletConnect from "./hooks/useWalletConnect";
+
+const StyledTitle = styled(Title)`
+  margin-top: 0;
+`;
+
+const StyledText = styled(Text)`
+  margin-bottom: 8px;
+`;
+
+const StyledHelpLink = styled.div`
+  display: flex;
+
+  > :first-of-type {
+    margin-right: 5px;
+  }
+`;
+
+const WCContent = styled.div`
+  min-width: 400px;
+  margin-right: 20px;
+`;
+
+const HelpLink = () => (
+  <StyledHelpLink>
+    <a
```

Comment:
you can just put the styles from `StyledHelpLink` to `a`, can't you?

Reply:
I think I can't. display: flex should be applied to the container.